handler.go

```diff
@@ -145,6 +145,16 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, u *User) {
 		return
 	}
 
+	// If authentication is enabled and there are no users yet, make sure
+	// the first statement is creating a new cluster admin.
+	if h.AuthenticationEnabled && h.server.UsersLen() == 0 {
```

Comment:
But where do you set it when you fire up the HTTP server?  That's the line I can't seem to find.  From what I can see, it's always going to be false, regardless of what you tell the config.